### ISSUE TITLE
[Accessibility][Keyboard -New Dialog]: Error should displayed as soon as the mandatory field are blank or duplication.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2074]
**User Experience** 
User with vision disability will not get any information on clicking the next button then they need to navigate back to the 'Name" edit field which consumes unnecessarily increases the user's effort. 

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to 'New Dialog' button available in the filter dialog pane and activate it..
4. Type the name of the dialog which is already created ex- Actions or leave it blank and navigate forward to click next button.
5. Observer the issue.

**Actual:** 
As the user navigate through the mandatory field and leave it blank no error message displayed until they press next.

**Expected:**
On leaving the mandatory field blank or duplication of the name, the error message should be displayed as soon as they press 'Tab' key.

**MAS Impacted:** [MAS3.3.4](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={5f773cee-c1c7-4292-b330-cb2f4aad00c4})

**Attachment:**
[MAS3.3.4_Error should displayed as soon as the mandatory field.zip](https://github.com/microsoft/BotFramework-Composer/files/4249129/MAS3.3.4_Error.should.displayed.as.soon.as.the.mandatory.field.zip)

**Comment by corinagum** [comment]
* note to fixer: typo on this dialogue - 'Duplicaton of dialog' should spell 'Duplication'

@ashish315, Is it required for a11y purposes that duplication checks happy before clicking a new button? I think it's a reasonable scenario to come across a dialog that runs checks *after* the submit and then provides feedback to the customer. 

For example, when creating a new username and password, often the customer isn't told until after they click submit that that username is already taken.

**Comment by ashish315** [comment]
@corinagum I'm totally agree with your comment even i cross checked the same with various Microsoft product and it seems like behavior. So, we are good to close this issue.

**Comment by ashish315** [comment]
@corinagum I'm totally agree with your comment.
But for the accessibility point of view its beneficial if the error or duplication check at the time of typing. You can take reference from 'SharePoint' one of the Microsoft product.

**Comment by corinagum** [comment]
@ashish315 That totally makes sense. Thanks for the feedback!

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
